Source: from Arch Linux
Upstream: yes, commit c0a141f
Reason: Fix build with newer GCC

diff --git a/src/util/pathname.hpp b/src/util/pathname.hpp
index c30df0a83..c7d1c7ca0 100644
--- a/src/util/pathname.hpp
+++ b/src/util/pathname.hpp
@@ -19,6 +19,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
 
 /** Simple class to allow a distinction of paths that refer to the
  *  filesystem and paths that refer to the datadir, it also hides
